docs/whats_new/v1.0.0.md

```diff
@@ -0,0 +1,305 @@
+
+Full Contributor List (ordered by # of commits):
+
+@shchur, @zhiqiangdon, @Innixma, @prateekdesai04, @FANGAreNotGnu, @yinweisu, @taoyang1122, @LennartPurucker, @Harry-zzh, @AnirudhDagar, @jaheba, @gradientsky, @melopeo, @ddelange, @tonyhoo, @canerturkmen, @suzhoum
+
+## Spotlight: Tabular Performance Enhancements
+
+AutoGluon 1.0 features **major enhancements to predictive quality**, establishing a new state-of-the-art in Tabular modeling.
+To the best of our knowledge, AutoGluon 1.0 marks the largest leap forward in the state-of-the-art for tabular data since the [original AutoGluon paper](https://arxiv.org/abs/2003.06505) from March 2020.
+The enhancements come primarily from two features:
+Dynamic stacking to [mitigate stacked overfitting](https://github.com/autogluon/autogluon/issues/2779#issuecomment-1736468165),
+and a new learned model hyperparameters portfolio via Zeroshot-HPO, obtained from the newly released [TabRepo](https://github.com/autogluon/tabrepo) ensemble simulation library.
+Together, they lead to a **75% win-rate compared to 0.8 with faster inference speed, lower disk usage, and higher stability.**
+
+### AutoML Benchmark Results
+
+OpenML released the [official 2023 AutoML Benchmark results](https://arxiv.org/abs/2207.12560) on November 16th, 2023.
+Their results show AutoGluon 0.8 as the state-of-the-art in AutoML systems across a wide variety of tasks:
+"Overall, in terms of model performance, AutoGluon consistently has the highest average rank in our benchmark."
+We now showcase that AutoGluon 1.0 achieves far superior results even to AutoGluon 0.8!
+
+Below is a comparison on the [OpenML AutoML Benchmark](https://openml.github.io/automlbenchmark/index.html) across 1040 tasks.
+LightGBM, XGBoost, and CatBoost results were obtained via AutoGluon, and other methods are from [the official AutoML Benchmark 2023 results](https://arxiv.org/abs/2207.12560).
+AutoGluon 1.0 has a 95%+ win-rate against traditional tabular models, including **a 99% win-rate vs LightGBM and a 100% win-rate vs XGBoost**.
+AutoGluon 1.0 has between an 82% and 94% win-rate against other AutoML systems.
+For all methods, AutoGluon is able to achieve >10% average loss improvement (Ex: Going from 90% accuracy to 91% accuracy is a 10% loss improvement).
+**AutoGluon 1.0 achieves first place in 63% of tasks**, with lightautoml having the second most at 12% (AutoGluon 0.8 previously took first place 48% of the time).
+AutoGluon 1.0 even achieves a 7.4% average loss improvement over AutoGluon 0.8!
+
+| Method                       | AG Winrate | AG Loss Improvement | Rescaled Loss |     Rank | Champion |
+|:-----------------------------|:-----------|:--------------------|--------------:|---------:|:---------|
+| AutoGluon 1.0 (Best, 4h8c)   | **-**      | **-**               |      **0.04** | **1.95** | **63%**  |
+| lightautoml (2023, 4h8c)     | 84%        | 12.0%               |           0.2 |     4.78 | 12%      |
```

Comment:
Can we include the version information for other libraries here, similar to what we do for AG, to maintain consistency?

Reply:
This is noted in the AMLB paper